CTFd/utils/exports/__init__.py

```diff
@@ -123,8 +118,7 @@ def import_ctf(backup, erase=True):
         )
 
     if erase:
-        drop_database()
-        create_database()
+        app.db.drop_all()
```

Comment:
While I was working on #1419, I experimented with this functionality to see if it resolved an issue I was running into but it didn't. This might be better practice but I am not convinced that it has an actual benefit in CTFd.